wallet-confs/zerozed--v0.13.1.conf

```diff
@@ -0,0 +1,13 @@
+server=1
+listen=1
+daemon=1
+gen=0
+rpcuser=
+rpcpassword=
+rpcallowip=127.0.0.1
+rpcport=2601
+port=2600
+txindex=2
```

Comment:
Usually a binary value. Are you sure it should be 2? (2 will probably work just as well as 1, but it looks a bit odd!)

Reply:
Thanks sir, looking at the wallet config there is no txindex=2 only in the xbridge.conf